<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fuzzymatchquery</class>
 <widget class="QWidget" name="fuzzymatchquery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模糊查询界面</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>1061</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Button_deposit_inquire">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>查询存款</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>71</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Button_draw_inquire">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>查询取款</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>71</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Button_transfer_inquire">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>查询转账</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>70</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Button_loan_inquire">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>查询贷款</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>71</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Button_payback_inquire">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>查询还款</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="inquire_condition">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>315</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>315</width>
     <height>50</height>
    </size>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>47</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>根据名字进行模糊查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>根据用户卡号进行模糊查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>根据日期进行模糊查询</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="inquire_content">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>459</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>560</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>确定查询</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="information_brower">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>811</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
